### PR TITLE
New governor/test with split propose queue

### PR DESCRIPTION
**Pull request by crazybuster**

### CODE REVIEW

File: contracts/contracts/governance/Governor.sol [hunk 1]
```diff
@@ -90,7 +87,7 @@ contract Governor {
         Proposal memory newProposal = Proposal({
             id: proposalCount,
             proposer: msg.sender,
-            eta: add256(block.timestamp, timelock.delay()),
+            eta: 0,
```
Comment (tomlinton) [inline contracts/contracts/governance/Governor.sol:90]: Do we want to keep this delay?

Comment (tomlinton) [reply contracts/contracts/governance/Governor.sol:90]: Nevermind I see it is handled below

**Review by tomlinton — APPROVED** [review]

**Review by franckc — APPROVED** [review]
A few minor comments otherwise LGTM - thanks for writing those scripts!

File: contracts/deploy/10_propose_governor.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  console.log("Running 10_propose_governor deployment...");
+
+  const sGovernor = ethers.provider.getSigner(governorAddr);
+  const sDeployer = ethers.provider.getSigner(deployerAddr);
+
+  // Deploy a brand new MinuteTimeLock and a new Governor
+  // This is timelock where the delay is only a minute
+  const dMinuteTimelock = await deploy("MinuteTimelock", {
+    from: deployerAddr,
+    args: [60], // needs to be a second different for it to deploy a new Timelock
```
[the remark this answers is not in the document]
Comment (crazybuster) [reply contracts/deploy/10_propose_governor.js:36]: skipIfAlreadyDeployed:false doesn't actually do anything, I basically had to change the contract to force it to deploy again. Might be a ganache issue.

**Pull request merged**